pkg/asset/cluster/quota.go

```diff
@@ -0,0 +1,29 @@
+package cluster
```

Comment:
This shouldn't live here as it's not a cluster asset. A cluster asset would be an asset that is a dependency for creating a cluster.

I don't think we should think of this file as an "[asset](https://github.com/openshift/installer/blob/master/docs/design/assetgeneration.md)". Although it should be written to the asset directory, as all logs, output, metadata is.

Better to move this to the destroy package.

Reply:
I put this here to mirror metadata. It's as much of an "asset" as metadata is, no?